test/unit/test_data_io.py

```diff
@@ -468,18 +499,19 @@ def test_get_training_data_iters():
     train_max_length = 30
     dev_line_count = 20
     dev_max_length = 30
-    expected_mean = 1.0
-    expected_std = 0.0
+    expected_mean = 0.9208152692746332
```

Comment:
Why do these values change? Does this test now run with prepended text always?

Reply:
That's right. Do you think it's better to parametrize this test with and without using prepended text?

Reply:
Given that without prepended text is probably the more prominent case, I would be in favor of parametrizing the test. Thanks!

Reply:
Done and made some minor changes (two commits). Could you approve again?